soroban-preflight/Cargo.toml

```
@@ -0,0 +1,23 @@
+[package]
+name = "soroban-preflight"
+description = "Soroban transaction preflight computations."
```

Suggest changed description to focus on the capability and less on one of the use cases we're using it in:
```suggestion
description = "Soroban host invocation simulations."
```